.editorconfig

```diff
@@ -2,18 +2,10 @@ root = true
 
 [*]
 indent_style = space
-indent_size = 2
```

Comment:
@corradio ups, you are right. I do this in a separate PR. 
The reason that I change the indent for html to 2 was because https://github.com/tmrowco/electricitymap/blob/master/mobileapp/www/index.html use mostly 2 spaces as indent. But it is a small file, and it is best to use 4 spaces for all files.